package.json

```diff
@@ -7,11 +7,13 @@
   "description": "A simple Node app built on Express, instantly up and running.",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js"
+    "start": "nodemon server.js"
```

Comment:
Love how you're using nodemon. Makes the life of any team member working with your repo so much easier. However, I must point out that nodemon wasn't installed as a project dependency 🤷‍♂️

Reply:
I think I have edited that line when I was trying to deploy the app to netlify .